app/enterprise/1.5.x/kong-for-kubernetes/deployment-options.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Kong for Kubernetes deployment options
+---
+
+## Deployment options
+
+Kong for Kubernetes provides two options for its Kong Enterprise container
+image:
+
+* [kong-enterprise-k8s][k8s-bintray] ([Kong for Kubernetes Enterprise][k4k8s-enterprise-install])
+* [kong-enterprise-edition][enterprise-bintray] ([Kong for Kubernetes with Kong
+  Enterprise][k4k8s-with-enterprise-install])
```

Comment:
Please flip these around, meaning, specify the product name first and in the brackets we can write (based on image).